pkg/operator/ceph/object/spec.go

```diff
@@ -154,6 +154,23 @@ func (c *clusterConfig) makeRGWPodSpec(rgwConfig *rgwConfig) (v1.PodTemplateSpec
 		ServiceAccountName: serviceAccountName,
 	}
 
+	if opsLogSidecar := c.store.Spec.Gateway.OpsLogSidecar; opsLogSidecar != nil {
+
+		// Generate ops log file name based on rgw daemon ID
+		opsLogFile := fmt.Sprintf("ops-log-%s", getDaemonName(rgwConfig))
+
+		// Add the side-car container named ops-log
+		shareProcessNamespace := true
+		podSpec.ShareProcessNamespace = &shareProcessNamespace
+		podSpec.Containers = append(podSpec.Containers, *controller.RgwOpsLogSidecarContainer(opsLogFile, c.clusterInfo.Namespace, *c.clusterSpec, opsLogSidecar.Resources))
+
+		if c.clusterSpec.LogCollector.Enabled {
```

Comment:
can we club this with line 174 checker

Reply:
@parth-gr we can but we'd still need to check for opslog enabled again, not sure if will make much difference then, let me know if i misunderstood anything